doc/whats_new/upcoming_changes/sklearn.inspection/302871.fix.rst

```diff
@@ -0,0 +1,3 @@
+- Fixes an issue where the partial dependence function accidentally tries to select positive
+class from binary predictions when they have already been selected.
+  By :user:`Stephen Pardy <stephenpardy>`
```

Comment:
```suggestion
- Fixes an issue where the partial dependence function accidentally tries to select
  positive class from binary predictions when they have already been selected.
  By :user:`Stephen Pardy <stephenpardy>`
```

Reply:
Actually by backporting this PR in 1.6 before the final release, we don't need to add any entry.
I added this regression in https://github.com/scikit-learn/scikit-learn/pull/30287 that is not release yet.